accord-core/src/test/java/accord/burn/random/RandomInt.java

```diff
@@ -0,0 +1,6 @@
+package accord.burn.random;
+
+public interface RandomInt
```

Comment:
C* simulator passes random in, where as BurnTest defines it on init... Personally I am a fan of the C* simulator's version but felt best to match existing code

Reply:
I think ideally we would move towards as similar a style as possible, and share as much code as possible between the two. But no need to undertake that now if you don't want.

Reply:
I don't actually mind the interfaces working that way, that doesn't force us to migrate 100% but does let us build swords that.

Ill try it out, I think it only impacts 1 real class and that can hold `Random` for now

Reply:
pushed